tests/conftest.py

```diff
@@ -134,3 +134,10 @@ def compare_records(actual: Drug, fixture: Drug):
     assert actual.approval_status == fixture.approval_status
     assert set(actual.other_identifiers) == set(fixture.other_identifiers)
     assert set(actual.xrefs) == set(fixture.xrefs)
+
+
+def compare_response(response: Dict, fixture: Drug, match_type: MatchType):
+    """Check that test response is correct."""
+    assert response['match_type'] == match_type
+    assert len(response['records']) == 1
```

Comment:
This may not always be one (I think). We should add a param for this and make the default 1.